compose/backend.yml

```diff
@@ -91,9 +91,9 @@ services:
       - service-auth-provider-api
 
   sidam-api:
-    image: hmcts.azurecr.io/hmcts/sidam/idam-api:1.0.0
+    image: "hmctspublic.azurecr.io/idam/api:pr-350"
     environment:
-      JAVA_OPTS: -Xms128m -Xmx512m -Dlogging.level.uk.gov.hmcts=DEBUG
+      JAVA_OPTS: -Xms128m -Xms512m -Dlogging.level.uk.gov.hmcts=DEBUG
```

Comment:
revert this otherwise @mario-paniccia  will after you 🏃

Reply:
Anand what's the rationale of this change please?

Reply:
@mario-paniccia  That change wasn't in master, when i updated yesterday morning
Did your changes made to master?